ui/core/src/actions/wallet.ts

```diff
@@ -30,6 +36,29 @@ export default ({
     async disconnect() {
       api.SifService.purgeClient();
     },
+    async connectToWallet() {
+      try  {
+        // TODO type
+        const address: any = await api.SifService.connect();
+        notify({
+          type: "success", 
+          message: "Sif Account connected", 
+          detail: address 
+        });
```

Comment:
Prob don't need this notify as there is one below listening for changes in connect.